File: 프로그래머스/2/42578. 의상/의상.java
import java.util.*;

class Solution {
    public int solution(String[][] cs) {
        int answer = 1;
        Map<String,Integer> m = new HashMap<>();
        
        for(String[] sarr : cs){
            m.put(sarr[1], m.getOrDefault(sarr[1] ,0) + 1);            
        }
        
        Iterator<Integer> iter = m.values().iterator();
        
        while(iter.hasNext()){
            answer *= iter.next().intValue() + 1;
        }
        
        return answer - 1;
    }
}

File: 프로그래머스/2/42578. 의상/의상.py
def solution(clothes):
    cloth_dict = {}

    for i in clothes:
        k = i[1]
        if k in cloth_dict:
            cloth_dict[k] += 1
        else:
            cloth_dict[k] = 1

    answer = 1
    for i in cloth_dict.values():
        answer *= i + 1
    
    return answer - 1